src/vaadin-grid.html

```diff
@@ -700,11 +715,13 @@
         this._getItem(index, row);
       }
 
+      /** @private */
       _columnTreeChanged(columnTree, splices) {
         this._renderColumnTree(columnTree);
         this.recalculateColumnWidths();
       }
 
+      /** @private */
       _renderColumnTree(columnTree) {
```

Comment:
Called from `vaadin-grid-column`, should be `@protected`?

Reply:
Done.